sdk/azidentity/go.mod

```diff
@@ -2,12 +2,13 @@ module github.com/Azure/azure-sdk-for-go/sdk/azidentity
 
 go 1.16
 
+replace github.com/Azure/azure-sdk-for-go/sdk/azcore => ../azcore
```

Comment:
Do we need this?

Reply:
Yes, until we release a version with `runtime.PipelineOptions`

Reply:
Ah I see.  So I assume we won't be releasing this azidentity change until that time.

Reply:
That's my plan. I thought, why write code that's already out of date vs. main when there's no urgency to release azidentity.